robotbase/src/main/java/com/team2357/lib/arduino/ArduinoSensor.java

```diff
@@ -0,0 +1,39 @@
+package com.team2357.lib.arduino;
+
+public class ArduinoSensor extends ArduinoUSBController{
+
+    private String m_deviceName;
+
+    public ArduinoSensor(String ttyDevice, String deviceName) {
```

Comment:
We do not want to take a device name here, we want this class to handle all three sensors. Taking and using the device name makes your three member methods return the same value off the same object, instead of the actual sensor they are referring to.